build/images/codegen/README.md

```diff
@@ -20,20 +20,21 @@ The image can only be built on an x86_64 machine (no arm support).
 
 Here is the table of codegen images that have been uploaded:
 
-| Tag                            | Change                                               |
-| :----------------------------- | ---------------------------------------------------- |
-| kubernetes-1.29.2-build.0      | Upgraded protoc (v26.0), protoc-gen-go (v1.33.0), protoc-gen-go-grpc (v1.3.0) |
-| kubernetes-1.29.2              | Upgraded K8s libraries to v1.29.2                    |
-| kubernetes-1.26.4-build.1      | Replace github.com/golang/mock with go.uber.org/mock |
-| kubernetes-1.26.4-build.0      | Upgraded Go to v1.21                                 |
-| kubernetes-1.26.4              | Upgraded K8s libraries to v1.26.4                    |
-| kubernetes-1.24.0-build.2      | Upgraded base image to ubuntu:22.04                  |
-| kubernetes-1.24.0-build.1      | Upgraded Go to v1.19                                 |
-| kubernetes-1.24.0-build.0      | Add controller-gen v0.9.0                            |
-| kubernetes-1.24.0              | Upgraded K8s libraries to v1.24.0                    |
-| kubernetes-1.21.0-build.1      | Upgraded protoc-gen-go to v1.5.2                     |
-| kubernetes-1.21.0-build.0      | Upgraded Go to v1.17                                 |
-| kubernetes-1.21.0              | Upgraded K8s libraries to v1.21.0                    |
-| kubernetes-1.19.8              | Upgraded K8s libraries to v1.19.8                    |
-| kubernetes-1.18.4              | Upgraded K8s libraries to v1.18.4                    |
-| kubernetes-1.17.6              | Upgraded K8s libraries to v1.17.6                    |
+| Tag                       | Change                                                                        |
```

Comment:
nit: for future reference, I try to avoid such formatting changes, as it creates an unnecessary diff to review, for no real added value.